src/Ticket.php

```diff
@@ -4758,9 +4777,14 @@ public static function showCentralList($start, $status = "process", bool $showgr
                         $options['criteria'][0]['value']      = CommonITILValidation::WAITING;
                         $options['criteria'][0]['link']       = 'AND';
 
-                        $options['criteria'][1]['field']      = 59; // validation aprobator
-                        $options['criteria'][1]['searchtype'] = 'equals';
-                        $options['criteria'][1]['value']      = Session::getLoginUserID();
+                        $options['criteria'][1]['criteria'][0]['field']      = 59; // validation aprobator
+                        $options['criteria'][1]['criteria'][0]['searchtype'] = 'equals';
+                        $options['criteria'][1]['criteria'][0]['value']      = Session::getLoginUserID();
+                        $options['criteria'][1]['criteria'][0]['link']       = 'AND';
+                        $options['criteria'][1]['criteria'][1]['field']      = 195; // validation aprobator substitute
+                        $options['criteria'][1]['criteria'][1]['searchtype'] = 'equals';
+                        $options['criteria'][1]['criteria'][1]['value']      = Session::getLoginUserID();
+                        $options['criteria'][1]['criteria'][1]['link']       = 'OR';
```

Comment:
Update the search criterias to show the tickets to validate, taking into account the substitutes